src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -426,17 +425,9 @@ private String getSamlRequestQueryStringSignature(String samlRequestQueryString)
         }
     }
 
-    private static DocumentBuilderFactory getDocumentBuildFactory() {
-        DocumentBuilderFactory documentBuilderFactory = DocumentBuilderFactory.newInstance();
-        documentBuilderFactory.setNamespaceAware(true);
-        return documentBuilderFactory;
-    }
-
     private static Element getMetadataDOM(final String xmlString) throws IOException, SAXException, ParserConfigurationException {
-        DocumentBuilder builder = null;
         try {
-            builder = documentBuilderFactory.newDocumentBuilder();
-            Document doc = builder.parse(new InputSource(new StringReader(xmlString)));
+            Document doc = Util.convertStringToDocument(xmlString);
```

Comment:
Should we use "Util.loadXML(...)" method instead ?
https://github.com/onelogin/java-saml/blob/master/core/src/main/java/com/onelogin/saml2/util/Util.java#L162

Reply:
Yes, looks like Util.loadXML method is better in this case.